crates/wasmparser/src/validator/operators.rs

```diff
@@ -121,6 +121,28 @@ pub enum FrameKind {
     ///
     /// This belongs to the Wasm exception handling proposal.
     TryTable,
+
+    #[cfg(feature = "legacy-exception-handling")]
+    /// A Wasm `deprecated-try` control block.
+    ///
+    /// # Note
+    ///
+    /// This belongs to the now deprecated Wasm exception handling proposal from Phase 1.
+    DeprecatedTry,
```

Comment:
Mind calling this "Legacy" instead of "Deprecated"?

Also mind updating the docs here to modernize what it points to?

Reply:
Updated ✅